scripts/historian.coffee

```diff
@@ -0,0 +1,15 @@
+# Description:
+# Sends all chat messages to Log Stash for archival.
+#
+# Commands:
+# No commands.
+#
+
+module.exports = (robot) ->
+  robot.hear /.*/i, (msg) ->
+    log = new Date() + ' | ' + msg.message.room + ' | ' + msg.message.user.name + ' | ' + msg.message
+    # msg.reply log
+    robot.http("http://192.168.99.100:56400")
+    .header('Content-Type', 'application/text')
+    .post(log) (err, res, body) ->
```

Comment:
Is the `(err, res, body) ->` supposed to be for the response? Can take out if you want to just ignore it?